### PR TITLE
feat(wallet): Create empty purses for issuers

### PR DESCRIPTION
**Pull request by samsiegart**
make-purse

### CODE REVIEW

**Review by michaelfig — COMMENTED** [review]
Looks good from a cursory glance.  Please fix the test failures and ping me again for review.

**Comment by samsiegart** [comment]
> Looks good from a cursory glance. Please fix the test failures and ping me again for review.

Sorry, fixed now

File: packages/dapp-svelte-wallet/react-ui/src/components/CardItem.jsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import './CardItem.scss';
 
-const CardItem = ({ children }) => {
+const CardItem = ({ children, hideDivider }) => {
   return (
     <div className="CardItem">
-      <div className="Divider" />
+      {!hideDivider && <div className="Divider" />}
```
Comment (michaelfig) [inline packages/dapp-svelte-wallet/react-ui/src/components/CardItem.jsx]: I have a mild preference for:

```suggestion
      {hideDivider || <div className="Divider" />}
```

File: packages/dapp-svelte-wallet/react-ui/src/views/Issuers.jsx [hunk 2]
```diff
@@ -1,7 +1,79 @@
...
+import CardItem from '../components/CardItem';
+import MakePurse from '../components/MakePurse';
+import { withApplicationContext } from '../contexts/Application';
 
-const Issuers = () => {
-  return <Typography variant="h1">Issuers</Typography>;
+import './Issuers.scss';
+
+// Exported for testing only.
+export const IssuersInternalDoNotImport = ({
```
Comment (michaelfig) [inline packages/dapp-svelte-wallet/react-ui/src/views/Issuers.jsx]: Can you rename these kinds of things to:
```suggestion
export const IssuersWithoutContext = ({
```

**Review by michaelfig — APPROVED** [review]
I'm really happy with this.  Good progress!

File: packages/dapp-svelte-wallet/react-ui/src/components/CardItem.jsx (hunk 1 above)
Comment (samsiegart) [reply packages/dapp-svelte-wallet/react-ui/src/components/CardItem.jsx]: SGTM, done

File: packages/dapp-svelte-wallet/react-ui/src/views/Issuers.jsx (hunk 2 above)
Comment (samsiegart) [reply packages/dapp-svelte-wallet/react-ui/src/views/Issuers.jsx]: Ah yea that's a bit better, done

**Pull request merged**